src/components/TodoList.tsx

```diff
@@ -0,0 +1,24 @@
+import { useContext } from 'react';
+import { TodosContext } from './TodosContext';
+import { TodoItem } from './TodoItem';
+
+export const TodoList: React.FC = () => {
+  const { todos, todosFilter } = useContext(TodosContext);
+
+  return (
+    <ul className="todo-list" data-cy="todosList">
+      {todos.filter((todo) => {
+        switch (todosFilter) {
+          case 'Active':
+            return !todo.completed;
+          case 'Completed':
+            return todo.completed;
+          default:
+            return true;
+        }
+      }).map((todo) => (
```

Comment:
better to create some filter function, and filter the array there, 'cause it looks too complex